container/libcontainer/handler.go

```diff
@@ -599,14 +600,16 @@ func setNetworkStats(libcontainerStats *libcontainer.Stats, ret *info.ContainerS
 	}
 }
 
-func newContainerStats(libcontainerStats *libcontainer.Stats, withPerCPU bool) *info.ContainerStats {
+func newContainerStats(libcontainerStats *libcontainer.Stats, withPerCPU, withDiskIO bool) *info.ContainerStats {
```

Comment:
Now that we are passing in two of these, I would rather just pass ignoreMetrics to the function.